content/articles/sharing-data-between-docker-containers/index.md

```diff
@@ -0,0 +1,201 @@
+### Sharing Data between Docker Containers
```

Comment:
No need to add hero on this file.

Reply:
You earlier said I remove it though @WanjaMIKE